cmd/checkheader/main.go

```diff
@@ -151,6 +166,24 @@ func tryConvertTypedef(w io.Writer, l string) bool {
 	return true
 }
 
+// tryConvertEnum adds a static check which verifies that
+// the enum contained in the line l
+// matches the corresponding OpenSSL value.
+// Only enum names starting with GO_ are converted.
+func tryConvertEnum(w io.Writer, l string) {
+	if !strings.HasPrefix(l, "GO_") {
+		return
+	}
+	if l[len(l)-1] == ',' {
+		l = l[:len(l)-1]
+	}
+	split := strings.SplitN(l, " = ", 2)
+	name := split[0][len("GO_"):]
+	fmt.Fprintf(w, "#ifdef %s\n", name)
+	fmt.Fprintf(w, "_Static_assert(%s == %s, \"%s\");\n", name, split[1], name)
```

Comment:
Should there be a bounds check for `split`? (In case of missing spaces in `" = "`, or auto-assigned enum entries?)

Reply:
Sure!